src/ripple/basics/SlabAllocator.h

```diff
@@ -214,95 +345,102 @@ class SlabAllocator
         @return a pointer to a block of memory from the allocator, or
                 nullptr if the allocator can't satisfy this request.
      */
-    std::uint8_t*
+    [[nodiscard]] std::uint8_t*
     allocate() noexcept
     {
-        auto slab = slabs_.load();
+        auto active = active_.load(std::memory_order::relaxed);
```

Comment:
May as well use acquire/release semantics for `active_` too. To be clear, I'll approve with `relaxed`, but on intel there won't be a perf difference so we may as well use the stronger semantics.